### ISSUE TITLE
Create elevator pitch and story document

### ISSUE THREAD

**Issue by rucuriousyet** [ExtremeInteractive/reboot#5]
We need resources that can be used to explain the basic (high level) story concept to someone who may be interested in the game, marketing the game, or joining our team etc. 

Additionally, a document (outline or similar) containing the major plot, characters and events would be great for us as a team to keep track of the story and its progression. Ideally, this should be a living (digital, continually updated) document) that we store somewhere in our Github organization.

**Comment by rucuriousyet** [comment]
Honestly, if you'd prefer, this issue could be broken into two separate cards...

**Comment by rucuriousyet** [comment]
Quoted from Slack: As the final Crew of Sim-scouts search mankind's reckless past for an alternate future, the remainder of humanity takes refuge in the center of the moon from the vicious monster, planet Earth, protected only by the surviving Keepers of the Cove.

**Comment by rucuriousyet** [comment]
The only thing I want to consider is changing "...center of the moon from the vicious monster, planet Earth..." to "...center of the moon from the vicious monster planet Earth has become...". @ourshadetree thoughts?

**Comment by ourshadetree** [comment]
I think that change makes sense.

**Comment by rucuriousyet** [comment]
Awesome!

**Comment by ourshadetree** [comment]
I tweaked the sentence order to keep it short: 

After Planet Earth becomes a vicious monster, the remainder of humanity takes refuge in the center of the moon while the final crew of Simulator-scouts searches mankind's reckless past for a new future.

**Comment by rucuriousyet** [comment]
That's damn good

-sent from my OnePlus X

**Comment by ourshadetree** [comment]
Excellent. Is this card finished then?

**Comment by rucuriousyet** [comment]
You know it

-sent from my OnePlus X

**Comment by rucuriousyet** [comment]
Yeah I moved it into the done column. I when you are ready to start
something else just pick a card thats assigned to you.
